atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -37,6 +37,12 @@
 from osbs.api import OSBS
 from osbs.exceptions import OsbsException
 
+have_get_orchestrator_build_logs = True
+try:
+    from osbs import get_orchestrator_build_logs
```

Comment:
This isn't a module-level function though, it's a method on a class instance (instances of osbs.api.OSBS).